src/AutomatedCar/Models/MatrixTwoByTwo.cs

```diff
@@ -0,0 +1,7 @@
+namespace AutomatedCar.Models
+{
+    /// <summary>This type will come from a library.</summary>
```

Comment:
Illetve az #54 alapján lehet, hogy nem is kell.

Reply:
Igen, én is így vagyok vele, még azért vizualizációtól feedback-re várok